### PR TITLE
ISC  - Custom erc20 - edits

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* removed link from sidebar category as it caused display issues at one point
* added ERC-native-token to the sidebar
* edits in text and format

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/build/isc/v1.0.0-rc.6/docs/how-tos/token/erc20-native-token.md [hunk 1]
```diff
@@ -8,25 +8,30 @@ tags:
...
 
-Once you [registered your native token as ERC20](./erc20-native-token.md) you can basically use it like any other ERC20 token, with functions like `transfer`, `balanceOf` and so on. But as the ERC20 token maps the native token on L2 there are some additional fetures.
+Once you [registered your native token as ERC20](./erc20-native-token.md) you can use it like any other ERC20 token, with functions like
+`transfer`, `balanceOf`, etc. But, as the ERC20 token maps the native token on L2, there are some additional ISC features 
+you can take advantage of.
 
 ## Example Code
 
-1. Get your `ERC20NativeTokens` instance (In this case we will use `erc20NativeTokensAddress` and the Foundry serial number to get the contract address)
+### Get Your `ERC20NativeTokens` Instance
```
Comment (Dr-Electron) [inline docs/build/isc/v1.0.0-rc.6/docs/how-tos/token/erc20-native-token.md:19]: I wouldn‘t add another title here. Especially as this is more about the nativeTokenID function of an ERC20NativeTokens object

File: docs/build/isc/v1.0.0-rc.6/sidebars.js [hunk 2]
```diff
@@ -159,8 +159,12 @@ module.exports = {
             {
               type: 'category',
               label: 'Token',
-              link: { type: 'doc', id: 'how-tos/token/introduction' },
               items: [
+                {
```
Comment (Dr-Electron) [inline docs/build/isc/v1.0.0-rc.6/sidebars.js:163]: You don‘t like linked categories? 😢 😆

Comment (lucas-tortora) [reply docs/build/isc/v1.0.0-rc.6/sidebars.js:163]: I do, but I'm certain we had an issue with them with this version of docusarus and I had to remove them

Comment (lucas-tortora) [reply docs/build/isc/v1.0.0-rc.6/sidebars.js:163]: We can leave them if you like, if the issue comes back we can remove it :)

Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/sidebars.js:163]: It was only related to use the introduction page both as the first page AND a category link. So both doesn't work.
Se this issue: https://github.com/iotaledger/hornet/issues/1825#issuecomment-1538487840

Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/sidebars.js:163]: So should we not use category pages? In the end I always saw category pages as something that you don't need while going through the sidebar but is nice to link to.

Comment (lucas-tortora) [reply docs/build/isc/v1.0.0-rc.6/sidebars.js:163]: I think we can use them, but not as a link.

Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/sidebars.js:163]: That is a no. Because if I talk about category page I mean using `link: ...`

**Pull request merged**